internal/compose/compose.go

```diff
@@ -21,15 +22,20 @@ func (ste *StateTransitionError) Error() string {
 
 // ImageBuild represents a single image build inside a compose
 type ImageBuild struct {
-	Id          int                    `json:"id"`
-	QueueStatus common.ImageBuildState `json:"queue_status"`
-	ImageType   common.ImageType       `json:"image_type"`
-	Manifest    *osbuild.Manifest      `json:"manifest"`
-	Targets     []*target.Target       `json:"targets"`
-	JobCreated  time.Time              `json:"job_created"`
-	JobStarted  time.Time              `json:"job_started"`
-	JobFinished time.Time              `json:"job_finished"`
-	Size        uint64                 `json:"size"`
+	Id          int               `json:"id"`
+	ImageType   common.ImageType  `json:"image_type"`
+	Manifest    *osbuild.Manifest `json:"manifest"`
+	Targets     []*target.Target  `json:"targets"`
+	JobCreated  time.Time         `json:"job_created"`
```

Comment:
Are these also backwards compat? Doesn't the job also store these?

Reply:
Very good point. Had to rework a bit, see below.